src/popup/styles/settings/_common.scss

```diff
@@ -10,3 +10,4 @@ $primary: #c32f34;
 
 $navbar-brand-font-size: $font-size-base;
 $nav-link-padding-y: 0;
+$navbar-nav-link-padding-x: 0.25rem;
```

Comment:
I don't see this being used 🔍

Reply:
It's a variable used by the [Bootstrap SCSS](https://github.com/twbs/bootstrap/blob/v4.5.3/scss/_variables.scss#L721).

Reply:
💡 oh, thanks!